datasets/sede/sede.py

```diff
@@ -0,0 +1,104 @@
+"""SEDE: Text-to-SQL in the Wild: A Naturally-Occurring Dataset Based on Stack Exchange Data."""
+
+
+import json
+
+import datasets
+
+
+logger = datasets.logging.get_logger(__name__)
+
+_CITATION = """\
+@misc{hazoom2021texttosql,
+      title={Text-to-SQL in the Wild: A Naturally-Occurring Dataset Based on Stack Exchange Data},
+      author={Moshe Hazoom and Vibhor Malik and Ben Bogin},
+      year={2021},
+      eprint={2106.05006},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """\
+SEDE (Stack Exchange Data Explorer) is new dataset for Text-to-SQL tasks with more than 12,000 SQL queries and their
+natural language description. It's based on a real usage of users from the Stack Exchange Data Explorer platform,
+which brings complexities and challenges never seen before in any other semantic parsing dataset like
+including complex nesting, dates manipulation, numeric and text manipulation, parameters, and most
+importantly: under-specification and hidden-assumptions.
+
+Paper (NLP4Prog workshop at ACL2021): https://arxiv.org/abs/2106.05006
+"""
+
+
+class SEDEConfig(datasets.BuilderConfig):
+    """BuilderConfig for SEDE."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for SEDE.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(SEDEConfig, self).__init__(**kwargs)
+
+
+class SEDE(datasets.GeneratorBasedBuilder):
+    """SEDE Dataset: A Naturally-Occurring Dataset Based on Stack Exchange Data."""
+
+    _DOWNLOAD_URL = "https://raw.githubusercontent.com/hirupert/sede/main/data/sede"
+    _TRAIN_FILE = "train.jsonl"
+    _VAL_FILE = "val.jsonl"
+    _TEST_FILE = "test.jsonl"
+
+    BUILDER_CONFIGS = [
+        SEDEConfig(
+            name="sede",
+            version=datasets.Version("1.0.0"),
+            description="SEDE Dataset: A Naturally-Occurring Dataset Based on Stack Exchange Data.",
+        ),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "QuerySetId": datasets.Value("string"),
```

Comment:
Isn't this field integer data type? If so, this would reduce the storage size.

Reply:
Yes, I just commit a fix. Thanks for noticing.